sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetReadSupport.scala

```diff
@@ -192,7 +194,8 @@ object ParquetReadSupport extends Logging {
       parquetSchema: MessageType,
       catalystSchema: StructType,
       caseSensitive: Boolean = true): MessageType = {
-    clipParquetSchema(parquetSchema, catalystSchema, caseSensitive, useFieldId = false)
+    clipParquetSchema(parquetSchema, catalystSchema, caseSensitive, useFieldId = false,
+      timestampNTZEnabled = true)
```

Comment:
We enable TIMESTAMP_NTZ here by default. @gengliangwang thoughts?

Reply:
Seems this method is not used?

Reply:
You are right, it appears to not being used. I removed that method.